cli/commands/find/find.go

```diff
@@ -21,6 +21,19 @@ import (
 
 // Run runs the find command.
 func Run(ctx context.Context, l log.Logger, opts *Options) error {
+	// Ensure WorkingDir is absolute to avoid filepath.Rel errors when comparing with absolute unit paths
+	if !filepath.IsAbs(opts.WorkingDir) {
+		absWorkingDir, err := filepath.Abs(opts.WorkingDir)
```

Comment:
I think this is wrong. We never want to care about the PWD of the process. We need the path to always be relative to the --working-dir flag.